### ISSUE TITLE
Rotas Fine Tunning foram alteradas, valide na documentação para ver se estou correto nas afirmações que postei abaixo

### ISSUE THREAD

**Issue by knrocha99** [landgraf-dev/openai-delphi#12]
OpenAiClient.pas

//  Request := CreateRequest('/fine-tunes', 'GET');
  Request := CreateRequest('/fine_tuning/jobs', 'GET');
  
//Request := CreateRequest('/fine-tunes/{fine_tune_id}/cancel', 'POST');
Request := CreateRequest('/fine_tuning/jobs/{fine_tuning_job_id}/cancel', 'POST');

Essas foram as duas necessidades que tive e notei que mudaram, porem não testei as demais, estou dando uma olhada para 
ver se encontro algo mais e volto a reportar.

Queria se possivel que montasse um demo :
1 -  como enviar audios e obter a resposta.
2 - Como fazer treinamentos via Fine tunning e como usar o modelo treinado, pois quando cria m fine tunning ele cria other models, queria saber como usar esse other model treinado com base nas regras enviadas para esse novo modelo.
4 - como enviar imagens

parabens por ter compartilhado esse fonte, me ajudou e com certeza ajudou muitos.

**Comment by wlandgraf** [comment]
Indeed, those are new endpoints for v2 of the API. We haven't updated the client yet for version 2.
